academy-of-management-review.csl

```diff
@@ -99,7 +102,14 @@
                 </date>
               </if>
             </choose>
-          </if>
+          </else-if>
+          <else-if variable="issued" match="any">
+            <choose>
```

Comment:
I think it's necessary. If there is no `issued`, `status` gets printed in its place. `status` shouldn't get printed twice in that case.